dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/LocalStorageOperatorFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.storage.hdfs;
+
+import org.apache.dolphinscheduler.plugin.storage.api.StorageOperate;
+import org.apache.dolphinscheduler.plugin.storage.api.StorageOperateFactory;
+import org.apache.dolphinscheduler.plugin.storage.api.StorageType;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(StorageOperateFactory.class)
+public class LocalStorageOperatorFactory implements StorageOperateFactory {
+
+    private static final String LOCAL_DEFAULT_FS = "file:///";
+
+    @Override
+    public StorageOperate createStorageOperate() {
+        HdfsStorageProperties hdfsStorageProperties = new HdfsStorageProperties();
+        hdfsStorageProperties.setDefaultFS(LOCAL_DEFAULT_FS);
```

Comment:
> If we set file:/// as the default fs, will the file store under /? 

No, it will not, and our base path still respect `defaultFS + RESOURCE_UPLOAD_PATH`, RESOURCE_UPLOAD_PATH default value is in `common.properties` config and its defalut value is `/dolphinscheduler`. So I think we can keep it in the way, WDYT

https://github.com/apache/dolphinscheduler/blob/b600234debd2fc01cf657b735b4864444207b502/dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java#L602-L610

Reply:
Yes, you are right.